app/models/item.rb

```diff
@@ -40,10 +40,10 @@ def self.filter_by_name(items, name)
 	end
 
 	def self.filter_by_min(items, min_price)
-		items.where("unit_price >= #{min_price}")
+		items.where("unit_price >= #{min_price}").order(:name)
 	end
 
 	def self.filter_by_max(items, max_price)
-		items.where("unit_price <= #{max_price}")
+		items.where("unit_price <= #{max_price}").order(:name)
```

Comment:
This appended code should reduce the erratic responses we were getting back from the tests.